install/apm/php/install.yml

```diff
@@ -8,19 +8,8 @@ target:
   destination: application
   os:
     - linux
-  context: |
-    ###### WARNING
-    Using the command will require a restart of php-fpm, nginx, or apache depending on the service that is detected. You will be given the option to allow the installer to perform this action for you, or you can restart manually.
-
-    ###### INFO
-    This single-command will instrument PHP applications on linux systems with php-fpm, nginx, or apache. For other environments, please see our PHP standard installation.
 
 install:
-  mode: targetedInstall
-  destination:
-    recipeName: php-agent-installer
-
-fallback:
```

Comment:
Same thing here - we should have `fallback` to the docs: https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php/#installation